libs/ui/src/components/atoms/Radio/Radio.styles.ts

```diff
@@ -0,0 +1,82 @@
+import styled from 'styled-components';
+import * as RadioGroupPrimitive from '@radix-ui/react-radio-group';
+
+import { Theme } from '../../../types/theming';
+
+export const Container = styled.div`
+  align-items: center;
+  display: flex;
+  padding-bottom: 14px;
+`;
+
+export const RadioGroup = styled(RadioGroupPrimitive.Root)``;
```

Comment:
Probably don't need the styled wrapper here. Can just assign RadioGroupPrimitive.Root to RadioGroup